Code/Framework/AzToolsFramework/AzToolsFramework/PaintBrushSettings/PaintBrushSettings.h

```diff
@@ -204,6 +220,19 @@ namespace AzToolsFramework
         //! Brush distance to move between stamps in % of paintbrush size. (25% is the default in Photoshop.)
         float m_distancePercent = 25.0f;
 
+        static inline constexpr size_t MinSmoothingRadius = 1;  // We need to use at least 1 pixel in each direction for smoothing
+        static inline constexpr size_t MaxSmoothingRadius = 10; // Limit the max to 10 pixels in each direction due to performance
+
+        //! Brush smoothing radius - the number of pixels in each direction to use for smoothing calculations.
+        size_t m_smoothingRadius = MinSmoothingRadius;
+
+        static inline constexpr size_t MinSmoothingSpacing = 0;     // A spacing of 0 means use adjacent pixels
+        static inline constexpr size_t MaxSmoothingSpacing = 50;    // Reasonable limit that doesn't spread the pixels too far out
```

Comment:
Are these max limits specific to terrain performance? Would it be beneficial to expose these as overridable by different paint brush types?

Reply:
They're not exactly specific to terrain, it's a bit hard for me to say if there's any benefit in exposing them per brush type. I'm inclined to leave them hard-coded for now, until we have a new brush type that clearly wants/needs different limits for these.

The smoothing radius limits here set our smoothing data queries to use anywhere from 9 - 441 data points per point we're smoothing. The smoothing spacing limits let us use a range from every point to every 50th point for querying for the smoothing, which also seems like a pretty big range.

It's possible that different underlying data types will want different ranges on these, but we can always expose these outwards later once we know for sure that different ranges are truly needed.